qos_intro.adoc

```diff
@@ -84,39 +84,38 @@ specification specifies:
 The capacity and bandwidth controller register interfaces for resource
 allocation and usage monitoring are defined as memory-mapped registers. Each
 controller that supports CBQRI provides a set of registers that are
-memory-mapped starting at an 8-byte aligned physical address. The memory-mapped
-registers may be accessed using naturally aligned 4-byte or 8-byte memory
+memory-mapped, starting at an 8-byte aligned physical address. The memory-mapped
+registers can be accessed by using naturally aligned 4-byte or 8-byte memory
 accesses. The controller behavior for register accesses where the address is not
 aligned to the size of the access, or if the access spans multiple registers, or
 if the size of the access is not 4 bytes or 8 bytes, is `UNSPECIFIED`. A 4-byte
 access to a register must be single-copy atomic. Whether an 8-byte access to a
-CBQRI register is single-copy atomic is `UNSPECIFIED`, and such an access may
+CBQRI register is single-copy atomic is considered `UNSPECIFIED`. This type of access can
 appear, internally to the CBQRI implementation, as if two separate 4-byte
-accesses were performed.
+accesses are performed.
 
 [NOTE]
 ====
-The CBQRI registers are defined in such a way that software can perform two
-individual 4 byte accesses, or hardware can perform two independent 4 byte
-transactions resulting from an 8 byte access, to the high and low halves of the
-register as long as the register semantics, with regards to side-effects, are
-respected between the two software accesses, or two hardware transactions,
+The CBQRI registers are defined so that the software can perform two
+individual 4 byte accesses or the hardware can perform two independent 4-byte
+transactions to the high and low halves of the register, as long as the register 
+semantics are respected between the two software accesses or two hardware transactions,
```

Comment:
@kersten1 hardware is allowed to perform two independent 4 byte transactions as a result of an 8 byte SW access. I think that clause was lost in the rewrite. It is also important to have the clause of honoring the side effects when making the transaction. Alternatively, that clause about side effect can be replaced by writing "...to the high and low halves, in that order, ...." as that will take care of the side effect.